CHANGELOG.md

```diff
@@ -11,6 +11,68 @@ All notable changes to Botonic will be documented in this file.
   </summary>
 </details>
 
+## [0.15.0] - 2020-26-10
+
+### Added
+
+- [Project](https://github.com/hubtype/botonic)
+
+  - New botonic.io [website](https://github.com/hubtype/botonic/pull/1014).
+
+* [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+
+  - New webchat features:
+
+    - Added new [`target`](https://github.com/hubtype/botonic/pull/970) prop to `Button` component to define the behavior to open links.
+    - Added new [`WhatsappTemplate`](https://github.com/hubtype/botonic/pull/972) component for Webchat.
+    - New [`getBotonicApp()`](https://github.com/hubtype/botonic/pull/1018) to access `Botonic` methods.
+    - Added a new webchat setting [`storageKey`](https://github.com/hubtype/botonic/pull/1019) that indicates the key name to use in order to store the webchat state in `localStorage` (or `sessionStorage`).
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+  - [Improvements](https://github.com/hubtype/botonic/pull/985) in Content.validate of text & shortText fields
+    - Validate text.text also if content has no keywords (it could be accessed through a button)
+    - Before, when shortText was empty, the contentful driver set it with the value of the name field (to ensure buttons did not show blank texts). Now this is managed by the Button model, so that MessageContent.validate() can report it.
+    - New [CMS method](https://github.com/hubtype/botonic/pull/1002) to deliver content from `id` when `type` is unknown.
+    - Improved [ContentsValidator](https://github.com/hubtype/botonic/pull/1001). Better logs through which you can get a list of errors.
```

Comment:
replace "Better logs through which you can get a list of errors." with "Now it's possible to report the detected errors into a callback".